templates/log4j.properties.j2

```diff
@@ -95,20 +95,34 @@ log4j.appender.cmd-logger.layout=org.apache.log4j.PatternLayout
 log4j.appender.cmd-logger.layout.ConversionPattern=%d{ISO8601} [%t] %-5p %c - %m%n
 
 #
-# server-logger - DailyRollingFileAppender
-# 
+# server-logger - RollingFileAppender
+#
 # Captures all output from the rundeckd server.
 #
-log4j.appender.server-logger=org.apache.log4j.DailyRollingFileAppender
-log4j.appender.server-logger.file=/var/log/rundeck/rundeck.log
-log4j.appender.server-logger.datePattern='.'yyyy-MM-dd
-log4j.appender.server-logger.append=true
-log4j.appender.server-logger.layout=org.apache.log4j.PatternLayout
-log4j.appender.server-logger.layout.ConversionPattern=%d{ISO8601} [%t] %-5p %c - %m%n
+log4j.appender.rollingFile=org.apache.log4j.RollingFileAppender
+log4j.appender.rollingFile.File=/var/log/rundeck/rundeck.log
+log4j.appender.rollingFile.layout=org.apache.log4j.PatternLayout
+log4j.appender.rollingFile.layout.ConversionPattern=%d{ISO8601} - %m%n
+log4j.appender.rollingFile.MaxFileSize=10MB
+log4j.appender.rollingFile.MaxBackupIndex=5
+log4j.appender.rollingFile.append=true
 
 #
-# audit 
-# 
+# server-logger - RollingFileAppender
+#
+# services.log
+#
+log4j.appender.rollingFile=org.apache.log4j.RollingFileAppender
+log4j.appender.rollingFile.File=$/var/log/rundeck/service.log
+log4j.appender.rollingFile.layout=org.apache.log4j.PatternLayout
+log4j.appender.rollingFile.layout.ConversionPattern=%d{ISO8601} - %m%n
+log4j.appender.rollingFile.MaxFileSize=10MB
```

Comment:
it would be nice parametrize `log4j.appender.rollingFile.MaxFileSize` and `log4j.appender.rollingFile.MaxBackupIndex`. We can change these values in the future, what do you think?

Reply:
What about 200mb rotating for each 7 days as default values?

Reply:
Well there is just two parameters like this which one I've updated and another one inserted.. So I'm gonna make some tests before to see if it works fine, but one ideia I had is to unify all the settings in the max of two settings and put rundeck.* to work with all the logs. 
- It will make more simple
- It is logs which with your suggestion of 7 days of retention it would be enough for all the logs.
-  It will make possible to rotate all the logs

So as you said it's tests might will be boring as I've to restart rundeck, so I'm gonna do its first test and make sure it works to go on to the next steps. 

what do you think @raphapr

Reply:
I see what you r saying but I'm afraid that violate the principle of ansible roles. A user may want to configure any log file through the role parameters, right?

Reply:
yeaah he will still be able to do that man.. right ? my proposal is to simplify the actual configs to all related logs.  `rundeck.jobs.log`, `rundeck.api.log`,`rundeck.access.log` etc.

so if anyone wants to create a new one it will be possible to create.

Anyway I don't know if it is possible and I am just gonna give it a try.